portfolio.html

```diff
@@ -0,0 +1,120 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link href="portstyle.css" type="text/css" rel="stylesheet">
+  
+</head>
+<body>
+
+	<section class="navigationBar">
+		<div class="navigationButtons">
+			<div class="home">
+			<a href="#">Home</a>
+		</div>
+			<div class="about">
+			<a href="#">About</a>
+		</div>
+			<div class="portfolio">
+			<a href="#">Portfolio</a>
+		</div>
+			<div class="socials">
+			<a href="#">Socials</a>
+		</div>
+		</div>
+		
+	</section>
+	
+	<section class="mainImage">
+		<div class="backgroundImage">
+		<div class="textoverlayed">
+		<div class="overlayText">
+			<h1>Marcus Bowyer</h1>
+		<div class="secondOverlayText">
+			<h2>Full Stack Web Devloper</h2>
+		</div>
+	</div>
+	</div>
+	</div>
+	</section>
+	
+	<section class="aboutMe">
+		<div class="aboutMeBox">
+		<h2 class="aboutMeTitle">About Me</h2>
+		<img src="https://ca.slack-edge.com/T3CSKGCFN-U019S5TT2A1-1182c80c547c-512" alt="#" class="pictureOfOwner">
+	</div>
+		<div class="textAboutMe">
+			<p>Hello my name is marcus and im a very dependable bloke you cna coinnt on me yes sir! SIKE get fucking Stick Bugged LOLjenagu nanrguao iraoubguoaerb nriopangpoeru nvrioaeungorae raueig rngioawngrpaiogn rnaignraeiogn rigoanoir rgao eiurbgo ragogaoie rng nrioagniroa ngraiognioraeng  rioangiaoerngrn gireang agnria ignarignro aignaioe nggnir ao;gnraiogngirona iong;raoi </p>
+	</div>
+	</section>
+		
+		<div class="portfolioTitle">
+			<h2>Portfolio</h2>
+		</div>
+		
+		<section class="threeBoxLayout">
+			<div class="leftBox">
+				<div class="leftImage">
```

Comment:
add width: 100%; to you leftImage css and I think you'll be a happier man!

Reply:
As spoken.